lib/src/sql/value/serde/ser/statement/live.rs

```diff
@@ -78,6 +80,9 @@ impl serde::ser::SerializeStruct for SerializeLiveStatement {
 			"archived" => {
 				self.archived = value.serialize(ser::uuid::opt::Serializer.wrap())?.map(Uuid);
 			}
+			"auth" => {
+				self.auth = None;
+			}
```

Comment:
why does auth have a special treatment here compared to the others above?

Reply:
The serde code here is for user input as opposed to our internal stored input. Stored structs work fine with the the [revisioned] macro. Because this path of execution would be user-led, we are safe to use None. Live query auth gets constructed internally in the system from the session context.